README.md

```diff
@@ -1,70 +1,256 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
- [ ] This is the default test that comes with React:


`src/App.test.js`:

![image](https://github.com/hernandanielzamora/Capstone-Front-End/assets/57421823/80bd3815-5a60-4b90-847e-3c711ea846e2)

![image](https://github.com/hernandanielzamora/Capstone-Front-End/assets/57421823/1c23882b-69e3-4d19-bbd6-87f1461b5f66)


Kindly remove it if not using it.  👍